README.md

```diff
@@ -1,366 +1,49 @@
-codec-marshaled data, and a Go value.
-
-It uses unmarshal to unmarshal the data into an interface{} value. It marshals
-the Go value using marshal, then unmarshals the result into an interface{}
-value.
-
-It then checks that the two interface{} values are deep-equal to one another,
-using CmpEquals(opts) to perform the check.
-
-See JSONEquals for an example of this in use.
-
-
-### Contains
-
-Contains checks that a map, slice, array or string contains a value. It's the
-same as using Any(Equals), except that it has a special case for strings - if
-the first argument is a string, the second argument must also be a string and
-strings.Contains will be used.
-
-For example:
-
-    c.Assert("hello world", qt.Contains, "world")
-    c.Assert([]int{3,5,7,99}, qt.Contains, 7)
-
-
-### ContentEquals
-
-ContentEquals is is like DeepEquals but any slices in the compared values will
-be sorted before being compared.
-
-For example:
-
-    c.Assert([]string{"c", "a", "b"}, qt.ContentEquals, []string{"a", "b", "c"})
-
-
-### DeepEquals
-
-DeepEquals checks that two arbitrary values are deeply equal. The comparison is
-done using the github.com/google/go-cmp/cmp package. When comparing structs, by
-default no exported fields are allowed. If a more sophisticated comparison is
-required, use CmpEquals (see below).
-
-Example call:
-
-    c.Assert(got, qt.DeepEquals, []int{42, 47})
-
-
-### Equals
-
-Equals checks that two values are equal, as compared with Go's == operator.
-
-For instance:
-
-    c.Assert(answer, qt.Equals, 42)
-
-Note that the following will fail:
-
-    c.Assert((*sometype)(nil), qt.Equals, nil)
-
-Use the IsNil checker below for this kind of nil check.
-
-
-### ErrorAs
-
-ErrorAs checks that the error is or wraps a specific error type. If so, it
-assigns it to the provided pointer. This is analogous to calling errors.As.
-
-For instance:
-
-    // Checking for a specific error type
-    c.Assert(err, qt.ErrorAs, new(*os.PathError))
-
-    // Checking fields on a specific error type
-    var pathError *os.PathError
-    if c.Check(err, qt.ErrorAs, &pathError) {
-        c.Assert(pathError.Path, Equals, "some_path")
-    }
-
-
-### ErrorIs
-
-ErrorIs checks that the error is or wraps a specific error value. This is
-analogous to calling errors.Is.
-
-For instance:
-
-    c.Assert(err, qt.ErrorIs, os.ErrNotExist)
-
-
-### ErrorMatches
-
-ErrorMatches checks that the provided value is an error whose message matches
-the provided regular expression.
+    qt.Assert(t, qt.Equals(someValue, wantValue))
 
-For instance:
+If you don’t want to abort on failure, use `Check` instead, which calls `Error`
+instead of `Fatal`:
 
-    c.Assert(err, qt.ErrorMatches, `bad wolf .*`)
+    qt.Check(t, qt.Equals(someValue, wantValue))
 
+The library provides some base checkers like `Equals`, `DeepEquals`, `Matches`,
+`ErrorMatches`, `IsNil` and others. More can be added by implementing the Checker
+interface.
```

Comment:
I like that the README is shorter now. The current package renders with quite a bit of duplication on pkg.go.dev, as each checker is documented twice - once in the README, and once in the actual docs.

Reply:
FWIW the reason for the original duplication was that much of the functionality was exported as global variables which don't get first-class treatment in the package docs, so the most important functionality was largely hidden. Now that everything is a function, we don't need that.